.pre-commit-config.yaml

```
@@ -71,7 +72,12 @@ repos:
     rev: "v2.5.1"
     hooks:
       - id: prettier
-        types: ["markdown"]
+        types_or: ["markdown", "json"]
```

It's nicer to have this diff-friendly:
```suggestion
        types_or:
        - markdown
        - json
```

---

ditto, if formatter didn't do it, i don't care about it.

---

Well, I do — lists are naturally multi-line entries.